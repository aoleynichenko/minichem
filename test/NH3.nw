start NH3
memory 400 mb
echo

geometry
N 0.000000000000000 0.000000000000000 0.284737340571879
H 0.469015890656038 0.812359352173423 -0.094912446857295
H 0.469015890656038 -0.812359352173423 -0.094912446857295
H -0.938031781312070 0.000000000000000 -0.094912446857295
end

basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
      0.16885540             0.44463454  
N    S
     99.1061690              0.15432897       
     18.0523120              0.53532814       
      4.8856602              0.44463454       
N    SP
      3.7804559             -0.09996723             0.15591627       
      0.8784966              0.39951283             0.60768372       
      0.2857144              0.70011547             0.39195739   
end

task scf
