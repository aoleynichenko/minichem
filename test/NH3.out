Minichem 0.0
By Alexander Oleynichenko, 2016
Build:      1281
Build date: Mar  9 2016 21:44:01
Compiler:   gcc 5.2.0
Date:       Wed Mar  9 22:43:17 2016
Cores:      1
Arguments:  ../bin/minichem NH3.nw 
------------------------------------------------------------
>>> NH3.nw
start NH3
memory 400 mb
echo

geometry
N 0.000000000000000 0.000000000000000 0.284737340571879
H 0.469015890656038 0.812359352173423 -0.094912446857295
H 0.469015890656038 -0.812359352173423 -0.094912446857295
H -0.938031781312070 0.000000000000000 -0.094912446857295
end

basis "ao basis" SPHERICAL
H    S
      3.42525091             0.15432897       
      0.62391373             0.53532814       
      0.16885540             0.44463454  
N    S
     99.1061690              0.15432897       
     18.0523120              0.53532814       
      4.8856602              0.44463454       
N    SP
      3.7804559             -0.09996723             0.15591627       
      0.8784966              0.39951283             0.60768372       
      0.2857144              0.70011547             0.39195739   
end

task scf
------------------------------------------------------------
                    Basis information
   H
  ----
  1   1S     3.42525091     0.15432897
             0.62391373     0.53532814
             0.16885540     0.44463454


   N
  ----
  1   1S    99.10616900     0.15432897
            18.05231200     0.53532814
             4.88566020     0.44463454

  2   2S     3.78045590    -0.09996723
             0.87849660     0.39951283
             0.28571440     0.70011547

  3   2P     3.78045590     0.15591627
             0.87849660     0.60768372
             0.28571440     0.39195739


Spherical basis set summary:
*---------*-----------------*--------*------------*
| Element |   Description   | Shells | Primitives |
*---------*-----------------*--------*------------*
|   H     | ao basis        |   1    |     3      |
|   N     | ao basis        |   3    |     9      |
*---------*-----------------*--------*------------*

------------------------------------------------------------
          ********************************
          *      Parallel SCF Module     *
          ********************************

Formula:   
Atoms:     4
Electrons: 10
Mol mass:  17.031
Charge:    0
Spin mult: 1
Units:     angstroms
------------------------------------------------------------
One-electron integrals evaluation algorithm: Obara-Saika
Two-electron integrals evaluation algorithm: Obara-Saika
One-electron integrals done in 0.000001 sec

****************** BASIS ORTHOGONALIZATION *******************
Algorithm: canonical
Basis functions elimination threshold: 1e-08
Overlap matrix lowest eigenvalue = 0.26751126
AO basis orthogonalization done in 0.000088 sec

Initial guess: extended Huckel theory
Initial energy = -67.58984518
Initial guess done in 0.003663 sec

#bfns = 8
#eris = 666

 iter.       Energy         Delta E       RMS-Dens       DIIS-Err     time
----------------------------------------------------------------------------
   1     -55.37366042     0.25755164     0.54998189     0.40186815    0.02
   2     -55.41632660    -0.04266617     0.22998175     0.03131309    0.05
   3     -55.46786874    -0.05154214     0.07079951     0.00269950    0.07
   4     -55.44854369     0.01932505     0.01516763     0.00041148    0.09
   5     -55.45429830    -0.00575461     0.00470003     0.00036081    0.12
   6     -55.45400928     0.00028902     0.00002724     0.00000941    0.14
   7     -55.45400914     0.00000013     0.00000019     0.00000003    0.16
----------------------------------------------------------------------------
          Total SCF energy =   -55.45400914
  Nuclear repulsion energy =    11.95863312


      Time for:           sec
---------------------------------
  Orthogonalization      0.000
  Initial guess          0.004
  Density matrix         0.028
  Fock matrix            0.114
  Diagonalization        0.000
---------------------------------


         Molecular Orbitals Summary
  +-----+-----+----------------+----------+
  | No  | Occ |     Energy     | Symmetry |
  +-----+-----+----------------+----------+
  | 1   |  2  |   -15.30435093 |     ?    |
  | 2   |  2  |    -1.09030018 |     ?    |
  | 3   |  2  |    -0.57295622 |     ?    |
  | 4   |  2  |    -0.57295622 |     ?    |
  | 5   |  2  |    -0.35219219 |     ?    |
  | 6   |  0  |     0.64023030 |     ?    |
  | 7   |  0  |     0.73159996 |     ?    |
  | 8   |  0  |     0.73159996 |     ?    |
  +-----+-----+----------------+----------+

        Mulliken population analysis
        ----------------------------

      Atom     Population     Charge
      1  N     7.47056941  -0.47056941
      2  H     0.84314353   0.15685647
      3  H     0.84314353   0.15685647
      4  H     0.84314353   0.15685647


        Loewdin population analysis
        ---------------------------

      Atom     Population     Charge
      1  N     7.30539157  -0.30539157
      2  H     0.89820281   0.10179719
      3  H     0.89820281   0.10179719
      4  H     0.89820281   0.10179719

               Multipole moments
               -----------------
  D       x            y            z
    -0.00000000   0.00000000  -0.70124707

  |D| = 0.70124707 a.u. = 1.78239209 Debye


Allocated: 48632 bytes
Freed:     43096 bytes
Max usage: 12384 bytes
Total time: 0.17 sec
